src/SimpleGraphs/simpledigraph.jl

```diff
@@ -26,6 +26,31 @@ end
 
 eltype(x::SimpleDiGraph{T}) where {T} = T
 
+# SimpleDiGraph(g)
+"""
+    SimpleDiGraph{T}(g::AbstractGraph)
```

Comment:
```suggestion
    SimpleDiGraph(g::AbstractGraph)
    SimpleDiGraph{T}(g::AbstractGraph)
```
Lets either add both signatures to the docstring, or mention that `T` is optional and what happens if it is omitted.